test/functional/mempool_packages.py

```diff
@@ -14,7 +14,7 @@
 class MempoolPackagesTest(BitcoinTestFramework):
     def set_test_params(self):
         self.num_nodes = 2
-        self.extra_args = [["-maxorphantx=1000"], ["-maxorphantx=1000", "-limitancestorcount=5"]]
+        self.extra_args = [["-maxorphantxsize=1000"], ["-maxorphantxsize=1000", "-limitancestorcount=5"]]
```

Comment:
1000 MB would be 1 GB, right? Does this not seem a little high (even for this test script)?

Reply:
Yes it is a "little" high, but 1000 orphan TX count was also high. The intent here was most likely to just disable the limit and keeping the number 1000 makes it easier later when conflicts arise (no need to think about that number)